src/defaults.js

```diff
@@ -1,9 +1,34 @@
 "use strict";
 const FormData = require("form-data");
+const parse = require("url-parse");
+const base32Decode = require("base32-decode");
 const { defaultOptions } = require("./utils");
 const { sign } = require("tweetnacl");
 const { toByteArray } = require("base64-js");
-const { MAX_REVISION } = require("skynet-js");
+const { MAX_REVISION, parseSkylink, JsonData, getFullDomainUrlForPortal } = require("skynet-js");
+
+const open = require("open");
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+const virtualConsole = new jsdom.VirtualConsole();
+virtualConsole.sendTo(console);
+require("global-jsdom")();
+global.document = new JSDOM(`...`, {
```

Comment:
What is this for? Needs a comment at the very least.

Reply:
Hello m-cut

When I take these out, a few functions no longer work.
It is not only used with "LoadMysky", but I can remove "loadMySky".
I only put "loadMySky" in your PR to see if you might be able to help, but if you can't, it's not that bad.

I only added "virtualConsole" for the "loadMySky" console.log. So I can take it out.